media/js/mozorg/history-slides.js

```diff
@@ -34,6 +34,7 @@
 
     // set up the slideshow
     // @requires: sequencejs
+    // NOTE: this version of sequence relies on deprecated jQuery methods
```

Comment:
Is this something that is likely to break in future jQuery updates? Wondering how we don't forget about it if so...

Reply:
Unsure here. I don't know how soon the jQuery team plans to completely remove the deprecated methods relied on by this library.

Could we perhaps write functional tests for this page that will fail if we update jQuery to a version that no longer supports the methods used in the sequence library?

Additionally, we could file a bug to update this page to a newer library (or maybe even a new design), making it a priority by it being a blocker for us to stay on the latest version of jQuery.

Reply:
Hmm, we do seem to have some [functional tests for this page](https://github.com/mozilla/bedrock/blob/master/tests/functional/test_history.py). Maybe that will be enough?

Either way, this is a very old page and probably in need of an update/redesign. Filing a bug may be worthwhile just to get it on the radar?

Reply:
Will file a bug 👍